app/javascript/components/SignUp.jsx

```diff
@@ -1,116 +1,86 @@
-          'access-token': response.request.getResponseHeader('access-token'),
-          'token-type': response.request.getResponseHeader('token-type'),
-          client: response.headers['client'],
-          expiry: response.headers['expiry'],
-          uid: response.headers['uid']
-        }));
-      sessionStorage.setItem('user_attributes', JSON.stringify(response.data.data));
-      this.props.history.push('/');
-    }).catch(error => {
-      this.setState({ errors: error.response.data.errors.full_messages });
-    });
-  };
+  render() {
+    const useStyles = makeStyles(theme => ({
+      root: {
+        flexGrow: 1,
+        backgroundColor: theme.palette.background.paper,
+      },
+    }));
 
-  handleChange = (user) => {
-    const currentUser = this.state.user;
-    currentUser[user.target.name] = user.target.value;
-    this.setState(currentUser);
-  };
+    function TabPanel(props) {
+      const { children, value, index, ...other } = props;
 
-  render() {
-    const { user } = this.state;
-    let errorMessages;
-    if (this.state.errors) {
-      errorMessages = <div>
-        {this.state.errors.map((error) => (
-          <p key={error.id}>{error}</p>
-        ))}
-      </div>;
+      return (
+        <Typography
+          component="div"
+          role="tabpanel"
+          hidden={value !== index}
+          id={`simple-tabpanel-${index}`}
+          aria-labelledby={`simple-tab-${index}`}
+          {...other}
+        >
+          <Box p={3}>{children}</Box>
+        </Typography>
+      );
     }
-    return (
-      <div>
-        <div className="errors">
-          {errorMessages}
-        </div>
-        <form onSubmit={this.handleSignUp}>
-          <div>
-            <label htmlFor="first_name">First name</label><br/>
-            <input type="text" name="first_name" value={user.first_name} onChange={this.handleChange}
-              className="form-control" />
-          </div>
 
-          <div>
-            <label htmlFor="last_name">Last name</label><br/>
-            <input type="text" name="last_name" value={user.last_name} onChange={this.handleChange}
-              className="form-control" />
-          </div>
+    TabPanel.propTypes = {
+      children: PropTypes.node,
+      index: PropTypes.any.isRequired,
+      value: PropTypes.any.isRequired,
+    };
 
-          <div>
-            <label htmlFor="email">E-mail</label><br/>
-            <input type="text" name="email" value={user.email} onChange={this.handleChange}
-              className="form-control" />
-          </div>
+    function a11yProps(index) {
+      return {
+        id: `simple-tab-${index}`,
+        'aria-controls': `simple-tabpanel-${index}`,
+      };
+    }
 
-          <div>
-            <label htmlFor="phone">Phone</label><br/>
-            <input type="text" name="phone" value={user.phone} onChange={this.handleChange}
-              className="form-control" />
-          </div>
+    const TabSign = () => {
```

Comment:
This is not clear

![image](https://user-images.githubusercontent.com/22749807/63544386-d2106200-c52d-11e9-90e8-bb8bf7c168c5.png)